kentikapi/internal/resources/base.go

```diff
@@ -44,13 +48,14 @@ func (b BaseAPI) PostAndValidate(ctx context.Context, url string, input interfac
 	if err := validation.CheckRequestRequiredFields("post", input); err != nil {
 		return err
 	}
-
+	LogPayload(b.LogPayloads, "Kentik API request - PostAndValidate", url, input)
 	payload, err := json.Marshal(input)
```

Comment:
I suggest to move logging request below json.Mashal() call. That way you can pass JSON string directly to the logPayload function and avoid unnecessary double-marshal.

Reply:
Done